test/multi/templates/header.htl

```diff
@@ -0,0 +1,5 @@
+<template data-sly-template.heading="${@ data}">
+    <h1>${data.title}</h1>
+    <sly data-sly-call="${separator}"/>
+</template>
+<template data-sly-template.separator>--------------------------------------</template>
```

Comment:
@jantimon Here, I added a test for a template that uses a template that is later defined in the same script. and it works now :-)

Reply:
Really cool 👍